From 6ea3d0c1ba707a386b134e20ed408d1de5ce91be Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marcus=20M=C3=BCller?= <marcus@hostalia.de>
Date: Wed, 21 Aug 2019 17:03:19 +0200
Subject: [PATCH 06/14] =?UTF-8?q?Revert=20"Prepared=20Changelog=20?=
 =?UTF-8?q?=E2=80=A6=20for=203.9"=20and=20"master:=20updating=20version=20?=
 =?UTF-8?q?to=20be=203.9.0.0-git"?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This reverts commit 3e6dae16d66dce84f7544cd7b12cc1401852c2f0.
This reverts commit f1475ce5eea60d4780621b5f62b8c7a226cd29a9.

This is the 3.8 release branch. These were stupid.
---
 CHANGELOG.md   | 2 --
 CMakeLists.txt | 4 ++--
 2 files changed, 2 insertions(+), 4 deletions(-)

diff --git a/CHANGELOG.md b/CHANGELOG.md
index 4d06c28d5e..495d821e48 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -7,8 +7,6 @@ Versioning](http://semver.org/spec/v2.0.0.html), starting with version 3.7.12.0.
 
 Older Logs can be found in `docs/RELEASE-NOTES-*`.
 
-## [3.9.0.0] - Unreleased
-
 ## [3.8.0.0] - 2019-08-09
 
 Witness me!
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0595755637..a47a945f9b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -48,9 +48,9 @@ message(STATUS "Build type set to ${CMAKE_BUILD_TYPE}.")
 
 # Set the version information here
 SET(VERSION_MAJOR 3)
-SET(VERSION_API   9)
+SET(VERSION_API   8)
 SET(VERSION_ABI   0)
-SET(VERSION_PATCH 0-git)
+SET(VERSION_PATCH 0)
 include(GrVersion) #setup version info
 
 # Minimum dependency versions for central dependencies:
-- 
2.20.1

